ddtrace/internal/module.py

```diff
@@ -90,12 +90,12 @@ def origin(module):
     # type: (ModuleType) -> str
     """Get the origin source file of the module."""
     try:
-        orig = abspath(module.__file__)  # type: ignore[type-var]
+        orig = abspath(object.__getattribute__(module, "__file__"))
```

Comment:
Can we add a comment about the `__getattribute__` usage? since we don't have a repro of the issue we have to be sure that this code doesn't regress